src/app/visual/infinite-scroll/infinite-scroll-visual.component.ts

```diff
@@ -6,4 +6,23 @@ import {
   selector: 'infinite-scroll-visual',
   templateUrl: './infinite-scroll-visual.component.html'
 })
-export class InfiniteScrollVisualComponent { }
+export class InfiniteScrollVisualComponent {
+  public firstList: any[] = [
+    0, 1
+  ];
+  public secondList: any[] = [
+    0, 1
+  ];
+
+  public secondEnabled = true;
+
+  public addToFirst() {
+    this.firstList.push(this.firstList.length);
+    this.firstList.push(this.firstList.length);
+  }
+
+  public addToSecond() {
+    this.secondList.push(this.secondList.length);
```

Comment:
- Wrap the contents of this method in a `setTimeout`.
- Serve the SPA and click the "Load more" button until there is a scrollbar.
- Slooooowly scroll until the infinite scroll's wait component is visible again.
- The infinite scroll adds around 2-3 dozen records instantaneously. This seems odd to me?

Reply:
```
public addToSecond() {
  setTimeout(() => {
    this.secondList.push(this.secondList.length);
    this.secondList.push(this.secondList.length);
  }, 1000);
}
```

Reply:
Here's the fun part: it seems to work okay when there is only ONE infinite scroll on the page.

Reply:
Hmm, I would say that it is against design to have two infinite scrolls in the same column like this visual test. Aside from that, perhaps we should be disabling the infinite scroll when there isn't a listener for `scrollEnd`. It doesn't really make sense to have an infinite scroll that never loads anything

Reply:
I'm going to redo this demo and visual tests to make more sense with actual usage